autoActivities.js

```diff
@@ -176,7 +176,7 @@ var AutoActivities = GObject.registerClass(
 		      if (this._settings.get_boolean("show-apps"))
 			      Main.overview.showApps();
 		      else
-			      Main.overview.secondWorkspaceIndexhow();
+			      Main.overview.show();
```

Comment:
Could you please inform better which buggy popup this would be and why the change?